pkg/appconsts/versioned_consts.go

```diff
@@ -1,12 +1,12 @@
 package appconsts
 
 import (
+	testground "github.com/celestiaorg/celestia-app/pkg/appconsts/testground"
 	v1 "github.com/celestiaorg/celestia-app/pkg/appconsts/v1"
-	v2 "github.com/celestiaorg/celestia-app/pkg/appconsts/v2"
 )
 
 const (
-	LatestVersion = v2.Version
+	LatestVersion = v1.Version
```

Comment:
[question] why was this downgraded from v2 to v1? Why is it v2 on main already?

Reply:
hmm no good catch that shouldn't have changed. maybe this was a messed up merge conflict or something